### ISSUE TITLE
爱可可微博热门分享(5.5)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#148]
No 1. <a href="https://weibo.com/1402400261/OcOP9sUZ7">[CL] RAG and RAU: A Survey on Retrieval-Augmented Language Model ......</a>
No 2. <a href="https://weibo.com/1402400261/OcRw2lxcW">开源的手持移动测绘系统，用于大规模3D地图的创建</a>
No 3. <a href="https://weibo.com/1402400261/OcK8WispW">Omniplex：致力于复现Perplexity的开源项目，专注于开发多功能的聊天界面和插件，使用 TypeScript 语言和 Next.js 框架</a>
No 4. <a href="https://weibo.com/1402400261/OcIIJuk7O">Secret Llama：完全私密的、无需服务器即可在浏览器中运行的大型语言模型(LLM)聊天机器人，支持 Mistral 和 LLama 3 等开源模型</a>
No 5. <a href="https://weibo.com/1402400261/OcOMxl0vo">[LG] Scalable Bayesian Inference in the Era of Deep Learning: ......</a>
No 6. <a href="https://weibo.com/1402400261/OcPiUuWtb">大语言模型在时间序列分析方面的表现</a>
No 7. <a href="https://weibo.com/1402400261/OcJHRcGLe">Lobe TTS：用于服务器和浏览器的高质量、可靠的文本到语音(TTS)/语音到文本(STT)库</a>
No 8. <a href="https://weibo.com/1402400261/OcRwlyXS1">crawlist：用于爬取网页列表的通用解决方案</a>
No 9. <a href="https://weibo.com/1402400261/OcPGV97lm">支持1M上下文的LLama-3 70B扩展模型，适用于长文本生成和对话场景</a>
No 10. <a href="https://weibo.com/1402400261/OczHeCo5F">大模型白盒子构建指南：旨在从零开始手写代码，深入理解并实践大型语言模型（LLM）的构建，包括模型结构、RAG、Agent和Eval任务</a>
No 11. <a href="https://weibo.com/1402400261/OcRMP9qLP">'LLM-And-More：一站式大模型训练及应用构建的解决方案，其覆盖了从数据处理到模型评估、从训练到部署、从想法到服务等整个流程' ......</a>
No 12. <a href="https://weibo.com/1402400261/OcOVrvRMW">[RO] What Foundation Models can Bring for Robot Learning in Manipulation ......</a>
No 13. <a href="https://weibo.com/1402400261/OcRJKwBep">Scribe：将音符渲染为 HTML 的工具</a>
No 14. <a href="https://weibo.com/1402400261/OcKtVEeeg">ollama-reply：开源的浏览器扩展，利用 AI 技术自动生成社交媒体互动回复，以促进社交网络增长</a>
No 15. <a href="https://weibo.com/1402400261/OcRm1e7kx">Datasets for MS MARCO Web Search：微软开源的大规模、信息丰富的网络数据集，包含数百万真实点击的查询-文档标签，用于模拟真实世界网页和查询分布，支持多种下游任务</a>
No 16. <a href="https://weibo.com/1402400261/OcORUbaId">[CL] LEGENT: Open Platform for Embodied Agents LEGENT是一个开放平台，通过构建交互式3D环境与数据生成管道，促进大规模语言模型与多模态模型的融合应用，以期望开发具有语言理解与交互能力的具身Agent。 ......</a>
No 17. <a href="https://weibo.com/1402400261/OcILm9tbl">sqlite-vec：一个正在开发中的 SQLite 扩展，提供了轻量、足够快的向量搜索功能，可以在任何支持 SQLite 的平台上运行</a>
No 18. <a href="https://weibo.com/1402400261/OcOZwqa8M">#免费##抽奖# 携手@博文视点Broadview 送出3本《大语言模型应用指南》</a>
No 19. <a href="https://weibo.com/1402400261/OcIQ7eAyh">Memory-Efficient Cross Entropy Loss：实现了一种线性投影加交叉熵损失的PyTorch模块，相比标准实现大幅降低了内存消耗</a>
No 20. <a href="https://weibo.com/1402400261/OcOuvvfTy">[CL]《When to Retrieve: Teaching LLMs to Utilize Information Retrieval Effectively》</a>
No 21. <a href="https://weibo.com/1402400261/OcK7JghYq">JAX-CORL：用 JAX 实现的离线强化学习算法的干净、单文件代码库，旨在提供高性能的基准测试和灵活性</a>
No 22. <a href="https://weibo.com/1402400261/OcINxErdP">DataScience_Interactive_Python：提供了一系列用于数据科学学习的交互式 Python dashboards</a>
No 23. <a href="https://weibo.com/1402400261/OcRNqBxKZ">检索增强语言模型(RALM)相关文献列表</a>
No 24. <a href="https://weibo.com/1402400261/OczhX3x8b">prettygraph：将文本转换为知识图谱的实验性UI</a>
No 25. <a href="https://weibo.com/1402400261/OczeoB4b3">spRAG：为非结构化数据设计的高性能RAG(Retrieval-Augmented Generation)框架，特别擅长处理复杂查询，如金融报告和法律文件</a>
No 26. <a href="https://weibo.com/1402400261/OczSpy9Mj">Time Series Analysis with Python：开源的 Python 时间序列分析课程，提供丰富的 Jupyter Notebooks和资料</a>
No 27. <a href="https://weibo.com/1402400261/OcK0D8di9">Green-Bit-LLM：用于微调、推理和评估 GreenBitAI 低bit语言模型（LLMs）的工具包</a>
No 28. <a href="https://weibo.com/1402400261/OcGugpFpV">机器遗忘(Machine Unlearning)概览</a>
No 29. <a href="https://weibo.com/1402400261/OcRBO4mdt">spaCyEx：spaCy扩展库，通过类似正则表达式的语法，简化了复杂模式匹配的创建</a>
No 30. <a href="https://weibo.com/1402400261/OcOuC1wz0">提出让大型语言模型自适应地判断何时需要调用信息检索系统来回答问题的训练方法，实验表明其优于固定策略。</a>
No 31. <a href="https://weibo.com/1402400261/OcRRydLgD">几篇论文实现代码</a>
No 32. <a href="https://weibo.com/1402400261/OcIJvnF8s">Blue Recorder：基于Green Recorder用Rust语言重写并改进的简单屏幕录制工具，专为Linux系统设计</a>
No 33. <a href="https://weibo.com/1402400261/OcIR78Xo5">'Kolmogorov-Arnold Q-Network (KAQN) - KAN applied to Reinforcement ......</a>
No 34. <a href="https://weibo.com/1402400261/Ocjk09QTv">Sequential Workflow Designer：一个无代码组件，用于构建基于流程的编程应用，完全通用，无需外部依赖，支持现代浏览器和移动设备</a>
No 35. <a href="https://weibo.com/1402400261/OcRoxgpxn">Rocketnotes：网页版 Markdown 笔记应用，具有大型语言模型(LLM)支持的文本补全、聊天和语义搜索功能</a>
No 36. <a href="https://weibo.com/1402400261/OcRLLbjpT">'RigelA - 用纯Rust编写的读屏(Screen Reader)项目，用于视力有障碍的人群操作电脑，软件会将屏幕上的各种信息转换成语音输出' ......</a>
No 37. <a href="https://weibo.com/1402400261/OcLA14tGU">晚安～ [月亮] ​</a>
No 38. <a href="https://weibo.com/1402400261/OczxipNgl">Knowledge Graph Builder App：用于从PDF文档构建知识图谱的应用，利用大型语言模型(LLM)和Neo4j数据库来组织和可视化信息</a>
No 39. <a href="https://weibo.com/1402400261/OcdI88DC9">《爱丽丝在可微奇境的冒险——第一卷，土地之旅》是一本关于现代(深度)神经网络设计的袖珍书，强调了构建高效处理N维数据模型的重要性，被人覆盖从卷积到Transformer、SSM 及其他一些主题</a>
No 40. <a href="https://weibo.com/1402400261/OcOZwqa8M">//@爱可可-爱生活：欢迎参与～//#免费##抽奖# 携手@博文视点Broadview 送出3本《大语言模型应用指南》</a>
No 41. <a href="https://weibo.com/1402400261/OcOsm4mnT">早！[太阳]</a>
No 42. <a href="https://weibo.com/1402400261/OcOvcsHGG">通过标准化评估框架展示LoRA微调语言模型的有效性，并探索了高效部署多个微调模型的方法。</a>
No 43. <a href="https://weibo.com/1402400261/OcRLdu0zH">go-size-analyzer：用于分析编译后的 Golang 二进制文件依赖的工具，帮助理解它们对最终构建大小的影响</a>
No 44. <a href="https://weibo.com/1402400261/OcOE4qGBj">[CV]《DOCCI: Descriptions of Connected and Contrasting Images》</a>
No 45. <a href="https://weibo.com/1402400261/OcOZfeC1K">今日推介(第1396期)：用多样化模型小组评估LLM生成、教LLM有效利用信息检索、310个与GPT-4竞争的微调LLM的技术报告、面向长程图像和视频生成的一致自注意力、关联/对比图像描述数据集 ......</a>
No 46. <a href="https://weibo.com/1402400261/OcOAWyWUS">[CV]《StoryDiffusion: Consistent Self-Attention for Long-Range Image and Video Generation》</a>
No 47. <a href="https://weibo.com/1402400261/OcOv8lEqL">[CL]《LoRA Land: 310 Fine-tuned LLMs that Rival GPT-4, A Technical Report》</a>
No 48. <a href="https://weibo.com/1402400261/Oc66q0f6i">'中文羊驼大模型三期项目 (Chinese Llama-3 LLMs) developed from Meta Llama ......</a>
No 49. <a href="https://weibo.com/1402400261/O6ilu3Wss">NLP入门指南：一文览尽自然语言处理的基本概念</a>
No 50. <a href="https://weibo.com/1402400261/OcOE7Bcr2">提出DOCCI，一个高质量的视觉语言数据集，其中包含精心设计的图像及对应的详尽人工描述，可有效评估和改进文本到图像生成模型。//[CV]《DOCCI: Descriptions of Connected and Contrasting Images》</a>
